ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectSSOTest.kt

```diff
@@ -109,20 +98,15 @@ class RedirectSSOTest : StringSpec() {
 
             // Get response from AuthnRequest
             val response = sendRedirectAuthnRequest(queryParams)
-            BindingVerifier.verifyHttpStatusCode(response.statusCode)
-
-            // Get response from plugin portion
-            val idpResponse = getServiceProvider(IdpSSOResponder::class)
-                    .getRedirectResponse(response).decorate()
-            // TODO When DDF is fixed to return a POST SSO response, uncomment this line
-            // SingleSignOnProfileVerifier.verifyBinding(idpResponse)
-            idpResponse.bindingVerifier().verify()
-
-            val responseDom = idpResponse.responseDom
-
-            CoreAuthnRequestProtocolVerifier(responseDom, REQUEST_ID, acsUrl[HTTP_POST],
-                    authnRequest.nameIDPolicy).verify()
-            SingleSignOnProfileVerifier(responseDom, acsUrl[HTTP_POST]).verify()
+            val finalHttpResponse =
+                    getServiceProvider(IdpSSOResponder::class).getResponseForRedirectRequest(
+                            response)
+//            SingleSignOnProfileVerifier.verifyBinding(finalHttpResponse)
```

Comment:
✏️ Should probably include a todo comment here indicating why this is commented out and when/how it will be enabled.